doc/source/user_guide/local-usage.rst

```diff
@@ -0,0 +1,95 @@
+.. _local_usage:
+
+.. currentmodule:: nomenclature
+
+Local usage of a project
+========================
+
+.. attention:: This page is intended for users who are familiar with Python and
+    `git <https://git-scm.com>`_ (or a service like `GitHub <https://github.com>`_).
+
+You can use the **nomenclature** package locally (on your machine) for validation
+and region-aggregration. This can be helpful as part of processing your model results,
+or to ensure that a submission to an IIASA Scenario Explorer instance will succeed.
```

Comment:
you sure?

Reply:
Had to think about it again myself and I'm pretty sure since it's about the sound of the following word and not the letter (https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an). But it kinda depends on how to pronounce IIASA ...

Reply:
Ah right, yes in that case you're correct. I was thinking of pronouncing it like "year" in which case "a" would be correct. In any case I'd say it's fine to leave as is.